views/jira-manual-app-creation.hbs

```diff
@@ -35,7 +35,14 @@
 
       <h1>Manual app Creation</h1>
```

Comment:
The title isn't 'Manual app creation' for the editing page. Per ticket description: `Manual app creation > Edit <github app name>` which comes from the Figma design.

<img width="559" alt="Screen Shot 2022-07-14 at 4 22 43 pm" src="https://user-images.githubusercontent.com/37155488/178914341-3554073d-830a-4538-8934-eb6a527d5334.png">

Reply:
Corrected